tests_e2e/tests/scripts/agent_persist_firewall-access_wireserver

```diff
@@ -90,15 +90,21 @@ echo "Finally online, Time: $(date --utc +%FT%T.%3NZ)"
 echo "Trying to contact Wireserver as $USER to see if accessible"
 
 echo ""
-echo "IPTables before accessing Wireserver"
-sudo iptables -t security -L -nxv -w
+echo "Firewall configuration before accessing Wireserver:"
+if command -v iptables; then
+  sudo iptables -t security -L -nxv -w
+else
+  sudo nft list table walinuxagent
+fi
 echo ""
 
 WIRE_IP=$(cat /var/lib/waagent/WireServerEndpoint 2>/dev/null || echo '168.63.129.16' | tr -d '[:space:]')
-if command -v wget >/dev/null 2>&1; then
-    wget --tries=3 "http://$WIRE_IP/?comp=versions" --timeout=5 -O "/tmp/wire-versions-$USER.xml"
-else
+if command -v curl >/dev/null 2>&1; then
```

Comment:
For some reason, on Azure Linux 3, wget returns success (0) on timeout. I am giving preference to curl instead.